src/rgw/rgw_rados.h

```diff
@@ -2595,10 +2582,6 @@ class RGWRados
 
   int create_pool(const rgw_pool& pool);
 
-  /**
-   * create a bucket with name bucket and the given list of attrs
-   * returns 0 on success, -ERR# otherwise.
-   */
   int init_bucket_index(RGWBucketInfo& bucket_info, int num_shards);
```

Comment:
@cbodley Yeah! At first observation, I thought of merging it. But why to drop, if the function still exist in the same file :-)